holdings_maintenance/pds4/pds4indexshelf.py

```
@@ -0,0 +1,499 @@
+    for path in args.table:
+
+        if not os.path.exists(path):
+
+            print('No such file or directory: ' + path)
+            sys.exit(1)
+
+        path = os.path.abspath(path)
+        pdsf = pdsfile.Pds4File.from_abspath(path)
+
+        if pdsf.isdir:
+            if not '/metadata/' in path:
```

Flake8 (7.2.0)

[error] test for membership should be 'not in'

(E713)